src/test/controllers/home.test.js

```diff
@@ -2,28 +2,14 @@ const test = require('tape'); //eslint-disable-line
 const request = require('supertest'); //eslint-disable-line
 const router = require('../../app.js');
 
-test('Test home route', (t) => {
+test('Test home route with loggedIn false', (t) => {
```

Comment:
We will need to look at how to run more comprehensive tests on the home route as a stretch goal...
Session-Cookie... 🍪